prdoc/pr_5130.prdoc

```diff
@@ -0,0 +1,39 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Add SingleAssetExchangeAdapter
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      SingleAssetExchangeAdapter is an adapter in xcm-builder that can be used
+      to configure the AssetExchanger in XCM to use pallet-asset-conversion.
```

Comment:
in reality with traits provided by `pallet-asset-conversion`, do not have it's impl

Reply:
What do you mean?
What I want to say is that with this new adapter, it's "easy" to setup XCM to work with `pallet-asset-conversion` to swap assets.

Reply:
I mean it can be configured with anything that implements SwapCredit and QuotePrirce